style/layer/highway_shield.js

```diff
@@ -6,7 +6,10 @@ function routeConcurrency(num) {
   return [
     "case",
     ["!=", ["get", "route_" + num], null],
-    ["image", ["concat", "shield_", ["get", "route_" + num]]],
+    [
+      "image",
+      ["concat", "shield\n", ["get", "route_" + num], "\n", ["get", "name"]],
+    ],
```

Comment:
The downside of introducing `name` into the image name is that the sprite sheet gets very fragmented by routes that travel over roads with many different names. The Kentucky parkways have uniform road names, but that certainly isn’t the case for other long-distance routes. We could conditionalize the addition of `name` on `route` equaling `US:KY:Parkway=`.

Reply:
I made it a `match` expression so it’s easy to add more special cases as necessary.